<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-18JS9SO" failures="1" tests="1" name="Default test(failed)" time="0.053" errors="0" timestamp="2023-11-03T11:38:22 EDT">
  <testcase classname="com.testng.practice.PracticeAssertions" name="getSum" time="0.012">
    <failure type="java.lang.AssertionError" message="expected [10] but found [8]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [8]
at com.testng.practice.PracticeAssertions.getSum(PracticeAssertions.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- getSum -->
  <testcase classname="com.testng.practice.PracticeAssertions" name="getMultiple" time="0.0">
    <ignored/>
  </testcase> <!-- getMultiple -->
</testsuite> <!-- Default test(failed) -->
